composer.lock

```diff
@@ -15811,5 +15811,5 @@
     "prefer-lowest": false,
     "platform": [],
     "platform-dev": [],
-    "plugin-api-version": "2.3.0"
+    "plugin-api-version": "2.2.0"
```

Comment:
I've noticed this changing in a few repos recently - don't think it's worth worrying about but I don't think it needs to be included in this PR (i.e. `composer install` runs fine in both cases).